hazelcast-sql/src/test/java/com/hazelcast/sql/impl/calcite/OptimizerInstantiationTest.java

```diff
@@ -0,0 +1,58 @@
+/*
```

Comment:
I did not create this class.
I'll remove it from this PR before merge if required